hoomd/hpmc/compute.py

```diff
@@ -311,16 +311,32 @@ def sdf(self):
         self._cpp_obj.compute(self._simulation.timestep)
         return self._cpp_obj.sdf
 
+    @log(category='sequence', requires_run=True)
+    def sdf_expansion(self):
+        """(*N_bins*,) `numpy.ndarray` of `float`): :math:`s[k]` - The scale \
+        distribution function :math:`[\\mathrm{probability\\ density}]`.
+
+        The :math:`x` at the center of bin :math:`k` is:
+        :math:`x = k \\cdot \\delta x + \\delta x/2`.
+
+        Attention:
+            In MPI parallel execution, the array is available on rank 0 only.
+            `sdf` is `None` on ranks >= 1.
```

Comment:
This appears to be copied and pasted from `sdf` without modification. Make the necessary changes to correctly document `sdf_expansion`.